src/buffers/RecordingBuffer.h

```diff
@@ -74,11 +74,13 @@ namespace timeshift {
       return PVR_ERROR_NO_ERROR;
     }
 
-    bool Open(const std::string inputUrl, const kodi::addon::PVRRecording& recording);
+    bool Open(const std::string inputUrl, const kodi::addon::PVRRecording& recording, int64_t streamId);
 
     std::atomic<bool> m_isLive;
 
     // recording start time
     time_t m_recordingTime;
+
+    int m_streamId;
```

Comment:
Where is this variable used?

Reply:
It is not used,  I was going to use it to retrieve and delete the map for single streams but I just used the current stream instead.

Reply:
Alright I removed that.

Reply:
Still not removed. Likely don't need changes in either `RecordingBuffer` file.